<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="1" failed="1" ignored="0" total="3" passed="1">
  <reporter-output>
    <line>
      <![CDATA[TestNG_ReportsAndLogs -> This runs once before class]]>
    </line>
    <line>
      <![CDATA[TestNG_ReportsAndLogs -> This runs before every method]]>
    </line>
    <line>
      <![CDATA[TestNG_ReportsAndLogs -> testMethod2]]>
    </line>
    <line>
      <![CDATA[TestNG_ReportsAndLogs -> This runs after every method]]>
    </line>
    <line>
      <![CDATA[TestNG_ReportsAndLogs -> This runs before every method]]>
    </line>
    <line>
      <![CDATA[TestNG_ReportsAndLogs -> testMethod1]]>
    </line>
    <line>
      <![CDATA[TestNG_ReportsAndLogs -> This runs after every method]]>
    </line>
    <line>
      <![CDATA[TestNG_ReportsAndLogs -> This runs once after class]]>
    </line>
  </reporter-output>
  <suite name="Regression TestSuite" duration-ms="203" started-at="2019-01-12T22:54:41Z" finished-at="2019-01-12T22:54:41Z">
    <groups>
    </groups>
    <test name="Application Test" duration-ms="203" started-at="2019-01-12T22:54:41Z" finished-at="2019-01-12T22:54:41Z">
      <class name="testclasses.TestNG_ReportsAndAnalog">
        <test-method status="PASS" signature="setUp()[pri:0, instance:testclasses.TestNG_ReportsAndAnalog@48140564]" name="setUp" is-config="true" duration-ms="47" started-at="2019-01-12T22:54:41Z" finished-at="2019-01-12T22:54:41Z">
          <reporter-output>
            <line>
              <![CDATA[TestNG_ReportsAndLogs -> This runs once before class]]>
            </line>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="PASS" signature="beforeMethod()[pri:0, instance:testclasses.TestNG_ReportsAndAnalog@48140564]" name="beforeMethod" is-config="true" duration-ms="0" started-at="2019-01-12T22:54:41Z" finished-at="2019-01-12T22:54:41Z">
          <reporter-output>
            <line>
              <![CDATA[TestNG_ReportsAndLogs -> This runs before every method]]>
            </line>
          </reporter-output>
        </test-method> <!-- beforeMethod -->
        <test-method status="FAIL" signature="testMethod2()[pri:0, instance:testclasses.TestNG_ReportsAndAnalog@48140564]" name="testMethod2" duration-ms="15" started-at="2019-01-12T22:54:41Z" finished-at="2019-01-12T22:54:41Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertTrue(Assert.java:44)
at org.testng.Assert.assertTrue(Assert.java:54)
at testclasses.TestNG_ReportsAndAnalog.testMethod2(TestNG_ReportsAndAnalog.java:36)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:73)
at org.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:123)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
            <line>
              <![CDATA[TestNG_ReportsAndLogs -> testMethod2]]>
            </line>
          </reporter-output>
        </test-method> <!-- testMethod2 -->
        <test-method status="PASS" signature="afterMethod()[pri:0, instance:testclasses.TestNG_ReportsAndAnalog@48140564]" name="afterMethod" is-config="true" duration-ms="0" started-at="2019-01-12T22:54:41Z" finished-at="2019-01-12T22:54:41Z">
          <reporter-output>
            <line>
              <![CDATA[TestNG_ReportsAndLogs -> This runs after every method]]>
            </line>
          </reporter-output>
        </test-method> <!-- afterMethod -->
        <test-method status="PASS" signature="testMethod1()[pri:0, instance:testclasses.TestNG_ReportsAndAnalog@48140564]" name="testMethod1" duration-ms="0" started-at="2019-01-12T22:54:41Z" finished-at="2019-01-12T22:54:41Z">
          <reporter-output>
            <line>
              <![CDATA[TestNG_ReportsAndLogs -> testMethod1]]>
            </line>
          </reporter-output>
        </test-method> <!-- testMethod1 -->
        <test-method status="PASS" signature="afterMethod()[pri:0, instance:testclasses.TestNG_ReportsAndAnalog@48140564]" name="afterMethod" is-config="true" duration-ms="0" started-at="2019-01-12T22:54:41Z" finished-at="2019-01-12T22:54:41Z">
          <reporter-output>
            <line>
              <![CDATA[TestNG_ReportsAndLogs -> This runs after every method]]>
            </line>
          </reporter-output>
        </test-method> <!-- afterMethod -->
        <test-method status="PASS" signature="beforeMethod()[pri:0, instance:testclasses.TestNG_ReportsAndAnalog@48140564]" name="beforeMethod" is-config="true" duration-ms="0" started-at="2019-01-12T22:54:41Z" finished-at="2019-01-12T22:54:41Z">
          <reporter-output>
            <line>
              <![CDATA[TestNG_ReportsAndLogs -> This runs before every method]]>
            </line>
          </reporter-output>
        </test-method> <!-- beforeMethod -->
        <test-method status="PASS" signature="cleanUp()[pri:0, instance:testclasses.TestNG_ReportsAndAnalog@48140564]" name="cleanUp" is-config="true" duration-ms="0" started-at="2019-01-12T22:54:41Z" finished-at="2019-01-12T22:54:41Z">
          <reporter-output>
            <line>
              <![CDATA[TestNG_ReportsAndLogs -> This runs once after class]]>
            </line>
          </reporter-output>
        </test-method> <!-- cleanUp -->
        <test-method status="SKIP" signature="testMethod3()[pri:0, instance:testclasses.TestNG_ReportsAndAnalog@48140564]" name="testMethod3" duration-ms="0" started-at="2019-01-12T22:54:41Z" depends-on-methods="testclasses.TestNG_ReportsAndAnalog.testMethod2" finished-at="2019-01-12T22:54:41Z">
          <exception class="java.lang.Throwable">
            <message>
              <![CDATA[Method TestNG_ReportsAndAnalog.testMethod3()[pri:0, instance:testclasses.TestNG_ReportsAndAnalog@48140564] depends on not successfully finished methods]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Throwable: Method TestNG_ReportsAndAnalog.testMethod3()[pri:0, instance:testclasses.TestNG_ReportsAndAnalog@48140564] depends on not successfully finished methods
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:888)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:73)
at org.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:123)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Throwable -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testMethod3 -->
      </class> <!-- testclasses.TestNG_ReportsAndAnalog -->
    </test> <!-- Application Test -->
  </suite> <!-- Regression TestSuite -->
</testng-results>
